lib/user_session_management.rb

```diff
@@ -174,8 +174,8 @@ def get_alternate_signup_url
 
   # New flow below
 
-    def save_unverified_user(user)
-      session[:unverified_user_id] = user.id
+    def save_unverified_user(user_id)
+      session[:unverified_user_id] = user_id
```

Comment:
why the change from passing the user object around to the id?

Reply:
It's an improvement. Passing the whole (giant and super important) User object is unnecessary and dangerous because it could be modified by the function.

Reply:
cool, just curious.. you should be able to merge now